A patch from Philip Guenther (procmail maintainer) fixing a
crash when procmail is invoked without arguments and then
receive a ^C.

--- src/comsat.c.orig	Tue Sep 11 06:55:46 2001
+++ src/comsat.c	Wed Dec  2 23:13:21 2009
@@ -120,7 +120,7 @@ void sendcomsat(folder)const char*folder;
 { int s;const char*p;
   if(!csvalid||!buf)		  /* is comat on and set to a valid address? */
      return;
-  if(!*cslgname||strlen(cslgname)+2>linebuf)	       /* is $LOGNAME bogus? */
+  if(!cslgname||!*cslgname||strlen(cslgname)+2>linebuf)/* is $LOGNAME bogus? */
      return;
   if(!(p=folder?folder:cslastf))		     /* do we have a folder? */
      return;
